Sources/BothamPullToRefreshHandler.swift

```diff
@@ -36,8 +39,14 @@ public class BothamPullToRefreshHandler: NSObject {
         self.refreshControl.endRefreshing()
     }
 
-    public func beginRefreshing(scrollView: UIScrollView) {
-        scrollView.setContentOffset(CGPoint(x: 0, y: -refreshControl.frame.size.height), animated: true)
-        self.refreshControl.beginRefreshing()
+    public func beginRefreshing() {
+        if !refreshControl.refreshing {
```

Comment:
I prefer to have the public API follow the Swift 3.0 guidelines.
https://swift.org/documentation/api-design-guidelines/